lava_test_plans/projects/lkft-android/include/metadata.jinja2

```diff
@@ -0,0 +1,22 @@
+  android.build: "{{ BUILD_NUMBER }}"
+  android.name: "{{ JOB_NAME }}"
+  android.url: "{{ REFERENCE_BUILD_URL }}"
+  android.version: "{{ ANDROID_VERSION }}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
```

Comment:
isn't 'ANDROID_'  prefix unnecessary ?

Reply:
ANDROID_ makes it clear this field is on ANDROID instead of on other like, BUILD, LAVA, etc.

Reply:
ok